app/src/main/java/com/martynaskairys/wallpee/testing/Alarm.java

```diff
@@ -0,0 +1,78 @@
+package com.martynaskairys.wallpee.testing;
+
+import android.app.AlarmManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Bundle;
+import android.os.PowerManager;
+import android.os.Vibrator;
+import android.telephony.TelephonyManager;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Toast;
+
+import com.martynaskairys.wallpee.ExitAppActivity;
+import com.martynaskairys.wallpee.MainActivity;
+import com.martynaskairys.wallpee.R;
+import com.martynaskairys.wallpee.RandomlyChangeWallpapers;
+import com.martynaskairys.wallpee.WallpaperService;
+
+public class Alarm extends BroadcastReceiver {
```

Comment:
to make the name of 'Alarm' more descriptive, I would rename it to AlarmBroadcastReceiver